module-2-html/exercises/06-top-5-songs/index.html

```diff
@@ -0,0 +1,69 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="author" content="Ervin Besic">
+    <meta name="description" content="Vjezba 06-top-5-songs">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Monthly Top 5 - Free Music Archive</title>
+</head>
+
+<body>
+    <h1>Free Music Archive</h1>
+    <h2>Monthly Top 5 (December 2019)</h2>
+    <h3>#1</h3>
```

Comment:
Osobno ne bih koristio `<h3>` elemente za numeriranje, nego `<ol>`. Ako bih koristio `<h3>`, onda bi to bilo za ime izvođača i pjesme (npr. "Ivo Ivić - Moja pjesma").